openSUSE/open-build-service
# Allow filtering by state in addition to involvement

Currently, the involvement filters only allow filtering with open state. We could make it possible for users to filter by just involvement, so that they could choose which state they want to filter by.

To test this:
1. Create a requests with a state other than open
2. Open a project/package that is a target or source of the request
3. See the request in both involvement: all and involvement: incoming or outgoing depending on source or target of the request
